content/en/OWNERS

```diff
@@ -8,6 +8,13 @@ reviewers:
 
 approvers:
 - sig-docs-en-owners
+- sig-docs-website-owners # important due to no_parent_owners option
 
```

Comment:
We could also add a `content/en/OWNERS/docs`:
```yaml
# Localization note: you do not need to copy this file into any target localization
approvers:
- sig-docs-en-owners

labels:
- sig/docs
```
so that sig-docs-website-owners aren't tagged there for review.

Reply:
Is it `content/en/OWNERS/docs` ?? I think it's Good idea!!

Reply:
https://github.com/kubernetes/website/pull/46423/commits/211604adb8e34c925ddf2d7553772e937079d8ac